staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_interval.go

```diff
@@ -140,6 +155,7 @@ func newCacheIntervalFromStore(resourceVersion uint64, store cache.Indexer, getA
 		}
 		buffer.endIndex++
 	}
+	sort.Sort(sortableWatchCacheEvents(buffer.buffer))
```

Comment:
nit: consider using the newer slices.Sort function, which runs faster.

Reply:
@wojtek-t mind checking if `slices.Sort` is faster ?

Reply:
ok, thanks.